.github/workflows/buildAndTestRyzenAI.yml

```diff
@@ -50,68 +50,45 @@ jobs:
       - name: Get mlir-aie
         id: clone-mlir-aie
         run: |
-          utils/clone-mlir-aie.sh
-          source air-venv/bin/activate
-
-          pushd mlir-aie
-          pip install -r python/requirements.txt
-          pip install -r python/requirements_ml.txt
-          HOST_MLIR_PYTHON_PACKAGE_PREFIX=aie pip install -r python/requirements_extras.txt
 
+          source air-venv/bin/activate
           VERSION=$(utils/clone-llvm.sh --get-wheel-version)
           pip -q download mlir==$VERSION \
             -f https://github.com/Xilinx/mlir-aie/releases/expanded_assets/mlir-distro
           unzip -q mlir-*.whl
           find mlir -exec touch -a -m -t 201108231405.14 {} \;
 
-          popd
+          MLIR_PYTHON_EXTRAS_COMMIT_HASH=$(utils/clone-mlir-aie.sh --get-mlir-python-extras-version)
+          HOST_MLIR_PYTHON_PACKAGE_PREFIX=aie python3 -m pip install git+https://github.com/makslevental/mlir-python-extras@$MLIR_PYTHON_EXTRAS_COMMIT_HASH -f https://github.com/llvm/eudsl/releases/expanded_assets/latest
+          
+          MLIR_AIE_VERSION=$(utils/clone-mlir-aie.sh --get-wheel-version)
+          python3 -m pip install mlir_aie==$MLIR_AIE_VERSION -f https://github.com/Xilinx/mlir-aie/releases/expanded_assets/latest-wheels/
 
-      - name: Build and Install mlir-aie
+      - name: Get llvm-aie
         run: |
           source air-venv/bin/activate
-          pushd mlir-aie
-
-          mkdir build
-          pushd build
-
-          export PATH=$VITIS/bin:$VITIS/aietools/bin:$PATH
```

Comment:
Vitis should get added back to the path. The chess tests never run...
```

-- Unable to find v++
-- Unable to find aiecompiler
-- Unable to find xchesscc
-- Unable to find aietools directory
...
-- Could NOT find AIETools (missing: AIETOOLS_DIR AIETOOLS_BINARY_DIR AIETOOLS_INCLUDE_DIR AIETOOLS_XCHESSCC AIETOOLS_XCHESS_MAKE AIETools_VERSION_MAJOR AIETools_VERSION_MINOR AIE AIE2) 
```
npu tests:
```
Total Discovered Tests: 48
  Unsupported: 21 (43.75%)
  Passed     : 27 (56.25%)
```

Reply:
> Do you suggest that we should support a super air build that can work with both chess and peano?

Yes, traditionally it has been the case that aircc can support both. Passing `--peano` to aiecc simply sets the peano install directory, so that is ok. aiecc uses the `--xchesscc, --no-xchesscc` and `--xbridge --no-xbridge` to control whether chess or peano is used for compiling and linking. aircc can select defaults for these based on how it's compiled or whatever, but aircc also includes `--xchesscc, --no-xchesscc` and `--xbridge --no-xbridge` flags which should be passed through to aiecc to override the defaults.

Reply:
Thanks for the detailed description. Now the CI lit tests are able to switch between peano and chess compiler backend, using one single AIR build. The tests are still being separated into `check-air-e2e-peano' and `check-air-e2e-chess' for convenience; with `check-air-e2e', both peano and chess tests shall run.